diff --git a/WORKSPACE b/WORKSPACE
index 4003e0e..c365ad6 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -5,14 +5,9 @@ load("//builders/bazel:deps.bzl", "python_deps")
 
 python_deps("//builders/bazel")
 
-http_archive(
+local_repository(
     name = "google_privacysandbox_servers_common",
-    # commit 2023-12-13
-    sha256 = "c45a8a7a714b5fc1f4e31a7fe6691eeee5817f91d29f786affc37fbeafc6e13d",
-    strip_prefix = "data-plane-shared-libraries-578c988ad077fa46056335fd07d316c26f452285",
-    urls = [
-        "https://github.com/privacysandbox/data-plane-shared-libraries/archive/578c988ad077fa46056335fd07d316c26f452285.zip",
-    ],
+    path = "data-plane-shared-libraries",
 )
 
 load(
